lib/src/async/string.dart

```diff
@@ -0,0 +1,21 @@
+// Copyright 2013 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'dart:convert' show Encoding, utf8;
+
+///  Converts a [Stream] of byte lists to a [String].
+Future<String> byteStreamToString(Stream<List<int>> stream,
```

Comment:
Do we even need it, given how short the implementation is? Maybe we should just deprecate and remove in 3.0.0?

Reply:
Good question -- it's used in Fuchsia in one location. It's also roughly zero-maintenance to keep around in async though.